### DISCUSSION TITLE
training with data from gaussian_splats_generate_training_data

### DISCUSSION THREAD

**Discussion by geoffpedder** [cgnomads/GSOPs#7] [General]
does data from gaussian_splats_generate_training_data work in jawshot postshot? I get an error when i try to import. Otherwise are there any tips you have for the training step in nerf studio

**Comment by david-rhodes** [comment]
Indeed, it does! 

The first thing you should check is that you're exporting images.bin, cameras.bin, and points3D.bin (**not** .ply). Then, drag your "images" and "sparse" directories together into the Postshot viewport. 

If that doesn't solve your issue, please post your error here.

**Comment by geoffpedder** [comment]
Is there somewhere i choose to output points3D.bin ?  When i click 'Save Training Data' i only get the .ply version

**Reply by geoffpedder** [reply]
nevermind, i wasn't on the develop branch. I see the option now.  Thanks for your help!

**Reply by david-rhodes** [reply]
Yes! Just make sure you are using the latest (from the `develop` branch), rather than the 1.0 release.
![image](https://github.com/user-attachments/assets/721b3d69-8fa4-463f-be5e-f03f1e158ad5)
